OpenRA.Mods.Common/Lint/CheckActorReferences.cs

```diff
@@ -39,7 +39,8 @@ void ILintServerMapPass.Run(Action<string> emitError, Action<string> emitWarning
 		void CheckTrait(Action<string> emitError, ActorInfo actorInfo, TraitInfo traitInfo, Ruleset rules)
 		{
 			var actualType = traitInfo.GetType();
-			foreach (var field in actualType.GetFields())
+			var members = actualType.GetFields().Union(actualType.GetProperties().Cast<MemberInfo>());
```

Comment:
- Suggest `Concat` since no need for de-duping `Union`.
- I think it'd be nicer to declare `IEnumerable<MemberInfo> members` rather than using `.Cast<>` but up to you.

Reply:
Removing the `.Cast<>` results in trying to concat a `FieldInfo[]` with a `PropertyInfo[]` and failing. I think I'm going to need the cast.